### DISCUSSION TITLE
SpringBoot Deploy

### DISCUSSION THREAD

**Discussion by HakanOZDABAK** [deta/space-docs#130] [Q&A]
Hello everyone, I wanna publish my Springboot backend program to Deta Space. Before that, I published my fastapi but I didn't find any documantation about Springboout deploying. Can you help me?

**Comment by lihugang** [comment]
See here.
https://deta.space/docs/en/build/quick-starts/custom
https://deta.space/docs/en/build/reference/spacefile#engine
https://deta.space/docs/en/build/reference/spacefile#run
